File: C++17/백준/Bronze/24087. アイスクリーム （Ice Cream）/アイスクリーム （Ice Cream）.cc
#include <iostream>
using namespace std;

int main () {
    int s, a, b;
    cin >> s >> a >> b;
    
    int total_price = 250;
    int current_h = a;
    
    if (a > s) {
        cout << 250;
    } else {
        while (current_h < s) {
            current_h += b;
            total_price += 100;
        }
        cout << total_price;
    }
    
    return 0;
}

File: Java/백준/Bronze/24087. アイスクリーム （Ice Cream）/アイスクリーム （Ice Cream）.java
import java.util.Scanner;

public class Main {
    public static void main (String[] args) {
        Scanner sc = new Scanner (System.in);
        
        int s = sc.nextInt();
        int a = sc.nextInt();
        int b = sc.nextInt();
        
        int total_price = 250;
        int current_h = a;
        
        if (a > s) {
            System.out.print(total_price);
        } else {
            while (current_h < s) {
                current_h += b;
                total_price += 100;
            }
            System.out.print(total_price);
        }
        
        sc.close();
    }
}